xrpl/models/transactions/permissioned_domain_delete.py

```diff
@@ -0,0 +1,41 @@
+"""Model for PermissionedDomainDelete transaction type."""
+
+import re
+from dataclasses import dataclass, field
+from typing import Dict, Final, Pattern
+
+from typing_extensions import Self
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
+
+_DOMAIN_ID_REGEX: Final[Pattern[str]] = re.compile("[A-F0-9]{64}")
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class PermissionedDomainDelete(Transaction):
+    """This transaction deletes a PermissionedDomain object."""
+
+    domain_id: str = REQUIRED  # type: ignore
+    """The domain to delete."""
+
+    transaction_type: TransactionType = field(
+        default=TransactionType.PERMISSIONED_DOMAIN_DELETE,
+        init=False,
+    )
+    """The transaction type (PermissionedDomainDelete)."""
```

Comment:
I don't think we've done this in other transactions but makes sense to add it for `transaction_type` and not exclude it from all other class variables.

Reply:
I don't understand. What do you propose to add/remove/change in this snippet?

Reply:
I agree with this change. I was just noting that we haven't added docstrings to `transaction_type` variables before but I do think we may as well such that all class variables have docstrings.

Reply:
yes okay 👍 
I felt we could add one for the sake of consistency